packages/server/graphql/uWSAsyncHandler.ts

```diff
@@ -0,0 +1,24 @@
+import {HttpResponse, HttpRequest} from 'uWebSockets.js'
+import getReqAuth from '../utils/getReqAuth'
+import safetyPatchRes from '../safetyPatchRes'
+import sendToSentry from '../utils/sendToSentry'
+
+export type uWSHandler = (res: HttpResponse, req: HttpRequest) => void
+const uWSAsyncHandler = (handler: uWSHandler, ignoreDone?: boolean) => async (
```

Comment:
pretty proud of this function. abstracts away all the pain of uWS so i can keep writing sloppy code & it just magically handles it. then tells me via sentry where the sloppy code is.